Source/Lib/TraktApiSharp/Authentication/TraktDeviceAuth.cs

```diff
@@ -232,7 +233,7 @@ public class TraktDeviceAuth
                 }
                 else if (responseCode == HttpStatusCode.BadRequest) // Pending
                 {
-                    await Task.Delay(device.IntervalInSeconds * 1000);
+                    await Task.Delay((int)device.IntervalInSeconds * 1000);
```

Comment:
I wonder if Task.Delay(TimeSpan.FromMinutes(1.7)); Will be more intuitive

Reply:
You're right, it would be more intuitive.
But it also introduces more overhead, since `Task.Delay(TimeSpan)` internally calls `Task.Delay(int milliseconds)` and also creates a `TimeSpan`-object.

[Source Reference](https://referencesource.microsoft.com/mscorlib/system/threading/Tasks/Task.cs.html#b070ae859dd3f7b7):
```csharp
public static Task Delay(TimeSpan delay, CancellationToken cancellationToken)
{
    long totalMilliseconds = (long)delay.TotalMilliseconds;
    if (totalMilliseconds < -1 || totalMilliseconds > Int32.MaxValue)
    {
        throw new ArgumentOutOfRangeException("delay", Environment.GetResourceString("Task_Delay_InvalidDelay"));
    }

    return Delay((int)totalMilliseconds, cancellationToken);
}
```